### ISSUE TITLE
non-esm version

### ISSUE THREAD

**Issue by stephenmathieson** [deepsweet/foxr#9]
Would you be up for publishing a version that doesn't require `esm`?

**Comment by deepsweet** [comment]
I knew this issue will come. I'll publish both CJS and ESM following the [Solving the "ESM in NodeJS" Odyssey](https://gist.github.com/WebReflection/a4e026e78af45ede8d0f7498cab44f91). I really like ESM only mode but this decision is not likely to become popular any time soon.

How painful is to use `esm` loader for 3rd party libs?

**Comment by stephenmathieson** [comment]
It's not terribly painful, just a little obnoxious since it's not part of my "regular" workflow. For now, I'm just doing this:

```
npm install --save-dev esm
npm install --save foxr
node -r esm my-program.js
```

This said, since adoption of ESM is pretty low, I do think it'd be great if you did publish a CJS version 😄

**Comment by deepsweet** [comment]
Just landed with v0.2.0:

```js
const foxr = require('foxr').default 

```

**Comment by stephenmathieson** [comment]
Thanks dude!
